coc-settings.json

```diff
@@ -39,9 +48,16 @@
     "python.linting.enabled": true,
     "python.linting.pylintEnabled": false,
     "python.linting.flake8Enabled": true,
+    "python.linting.flake8Args": ["--max-line-length", "88"],
```

Comment:
Sorry but not sorry 🕺🏻

Reply:
Why would you need this? `flake8` should be read from config file instead using `.flake8`

This way it can be overriden per project/user

https://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations